site/migration/01-migration.md

```diff
@@ -0,0 +1,257 @@
+| kpt cfg delete-subst DIR/ subst_name [flags]                                                     | kpt fn eval --image gcr.io/kpt-fn/search-replace:0.1 -- 'by-value=nginx:1.7.1' put-comment=''                                                                                                                                                    |
+| kpt cfg annotate DIR/STDIN [flags]                                                               | kpt fn eval --image gcr.io/kpt-fn/set-annotation:v0.1 -- 'name=foo' 'value=bar'                                                                                                                                                                  |
+| kpt cfg grep DIR/STDIN [flags]                                                                   | kpt fn eval --image gcr.io/kpt-fn/search-replace:v0.1 -- 'by-value=foo' 'by-path=bar'                                                                                                                                                            |
+| kpt cfg count DIR/STDIN [flags]                                                                  | Deprecated.                                                                                                                                                                                                                                      |
+
+### Kptfile schema changes
+
+The existing `v1alpha1` Kptfile format/schema is not compatible with kpt `v1.0`.
+Here is the schema for [v1alpha2 Kptfile] which is compatible with kpt `v1.0`.
+
+1. The `packageMetaData` section in `v1alpha1` Kptfile is transformed to `info` section in `v1alpha2` Kptfile.
+2. `upstream` section, in the `v1alpha1` Kptfile is split into `upstream` and `upstreamLock`
+   sections in `v1alpha2` Kptfile. `upstream` section can be modified by users to declare
+   the desired version of the package, `upstreamLock` is the resolved package state
+   and should not be modified by users.
+3. The `OpenAPI` section in `v1alpha1` Kptfile is deprecated. [Details below](#changes-to-setters).
+4. `dependencies` section in `v1alpha1` Kptfile is deprecated. [Details below](#kpt-pkg-sync-deprecated).
+5. `functions` section in `v1alpha1` Kptfile holds definitions for Starlark functions only.
+   This section is deprecated and all the functions can be declared in the `pipeline`
+   section including [Starlark functions].
+6. `inventory-object` is moved to `inventory` section in `v1alpha2` Kptfile. [Details below](#changes-to-live).
+
+### `kpt pkg sync` deprecated
+
+Nothing to worry about. `sync` is just a declarative version of `update` functionality.
```

Comment:
Wording suggestion:
'sync' functionality has been moved to the 'update` command.

Reply:
Good point. Update to `sync` merged with `update`